app/models/concerns/netlify_content.rb

```diff
@@ -0,0 +1,29 @@
+module NetlifyContent
+  extend ActiveSupport::Concern
+
+  class_methods do
+    def find_by_path(path, base:, try_index:)
+      dirname = Pathname(path).dirname
+      filename = Pathname(path).basename(".md")
+      full_path = base.join dirname, "#{filename}.md"
+      if File.exist?(full_path)
+        new full_path, base: base
+      elsif try_index
+        find_by_path(dirname.join(filename, "index.md"), base: base, try_index: false)
+      else
+        Rails.logger.error "Failed to find NetlifyContent path: #{full_path}"
+        fail NotFound
+      end
+    end
+  end
+
+  included do
+    attr_reader :parsed_file
+  end
+
+  def rendered_content
+    Kramdown::Document.new(parsed_file.content).to_html.html_safe
```

Comment:
I tweaked this a little with the video stuff--can you merge/rebase and get this to use the custom Kramdown parser (I think just `Kramdown::Document.new(parsed_file.content, input: "CustomParser")`